users/validators.py

```diff
@@ -4,14 +4,15 @@
 MAX_YEARS_OF_EXPERIENCE = 100
 
 
-class ProfileValidator(object):
+class ProfileValidator:
     @staticmethod
     def validate_experience(rq_data):
```

Comment:
Probably signature `def validate(rq_data)` would be a bit more useful due to unified interface